internal/fingerprint.go

```diff
@@ -29,57 +29,76 @@ type fingerprintRunner struct {
 	dataSender    network.DataSender
 	templateStore loggers.TemplateStore
 	logger        zerolog.Logger
+	// state tracking
+	failureCount int
+	// allTimeFailureCount is loaded from disk from last boot - used to determine edge-logging need. Increments by 1 per boot cycle even if retried many times in one boot
+	allTimeFailureCount int
```

Comment:
What is the primary reason for having all TimeFailureCount? Is it - send to edge logs first time VIN failure happens?